runtime/onert/backend/gpu_cl/open_cl/OpenclWrapper.cc

```diff
@@ -108,23 +108,34 @@ absl::Status LoadOpenCL()
 #endif
 }
 
+void UnloadOpenCL(void *libopencl)
+{
+  if (libopencl)
+  {
+    dlclose(libopencl);
```

Comment:
have you tested this code?

Reply:
TC worked... So i uploaded a commit.
But when I check, it doesn't come in.
So.. i will check more.. Thanks!!